runners/gearpump/pom.xml

```diff
@@ -69,6 +69,7 @@
                   <dependenciesToScan>
                     <dependency>org.apache.beam:beam-sdks-java-core</dependency>
                   </dependenciesToScan>
+                  <argLine>-noverify</argLine>
```

Comment:
After recent changes on Gearpump build, launching JVM will throw verify errors. I haven't found the root cause and this is a work around.